### ISSUE TITLE
how install

### ISSUE THREAD

**Issue by hkjalaan** [doadam/ziVA#2]
hi there how i can install it??

**Comment by arinc9** [comment]
write cd <exploit's location in your mac>
write make

**Comment by hkjalaan** [comment]
Creating directories
Beginning release build
Compiling: apple_ave_pwn.m -> build/release/apple_ave_pwn.o
g++: error: arm64-apple-darwin9: No such file or directory
g++: error: unrecognized command line option ‘-target’; did you mean ‘-ftarget=’?
Makefile:213: recipe for target 'build/release/apple_ave_pwn.o' failed
make[1]: *** [build/release/apple_ave_pwn.o] Error 1
Makefile:138: recipe for target 'release' failed
make: *** [release] Error 2

**Comment by stek29** [comment]
Try with clang

**Comment by hkjalaan** [comment]
clang: error: no input files

**Comment by stek29** [comment]
I meant try to install clang and then run make.
Not to offend, but why do you even need this, if you don't know what Makefile is?

**Comment by hkjalaan** [comment]
Its exploit iphone?
I HAVE INSTALL IT BUT THE SAME THING

بتاريخ ٢٨‏/٠٨‏/٢٠١٧ ٦:٣٠ ص، كتب "Viktor Oreshkin" <notifications@github.com

**Comment by stek29** [comment]
Yes, it is an iOS exploit PoC. And that's exactly why you most likely don't need it.
